saleor/graphql/product/types/categories.py

```diff
@@ -60,6 +61,11 @@ class Category(ModelObjectType[models.Category]):
             f"{DEPRECATED_IN_3X_FIELD} Use the `description` field instead."
         ),
     )
+    updated_at = graphene.DateTime(
+        required=True,
+        description="The date and time when the product was last updated."
```

Comment:
I just found small error here: 
```suggestion
        description="The date and time when the category was last updated."
```
This will require rebuilding the graphql.schema.

Reply:
@maarcingebala Sorry for the simple mistake. It has been corrected and `graphql.schema` has been rebuilt.